pkg/build/generator/generator_test.go

```diff
@@ -741,7 +741,7 @@ func TestGenerateBuildFromConfig(t *testing.T) {
 	bc := &buildapi.BuildConfig{
 		ObjectMeta: kapi.ObjectMeta{
 			Name:      "test-build-config",
-			Namespace: "test-namespace",
+			Namespace: kapi.NamespaceDefault,
```

Comment:
> why?

The new testing mock used by generated fake clients requires that namespaced resources come in with namespaces.  It doesn't understand defaulting.